io.github.seadve.Delineate.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "io.github.seadve.Delineate",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.node20"
+    ],
+    "command": "delineate",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
```

Comment:
Its harmless, but this seems to be a copy paste of directories that have nothing to do with this package.

Reply:
Ooh, I copied with other Rust app I published. I'm gonna check.

Reply:
Amended and it seems those files/directories do not actually exist in the build, as checked on the `builddir` directory from the following command:

```bash
flatpak run org.flatpak.Builder --force-clean --sandbox --user --install --install-deps-from=flathub --ccache --mirror-screenshots-url=https://dl.flathub.org/media/ --repo=repo builddir <manifest>
```